airbyte-metrics/reporter/src/main/resources/micronaut-banner.txt

```diff
@@ -1 +1,8 @@
-: airbyte-metrics-reporter :
+
```

Comment:
No strong opinon here but seeing the Airbyte banner, was the sign for me that my local deployment can be use. Having many of them will be confusing. If we decide on keeping them we should add it to the cron app as well.